runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollOverlay.java

```diff
@@ -43,9 +45,16 @@
 import net.runelite.client.ui.overlay.components.LineComponent;
 import net.runelite.client.ui.overlay.components.PanelComponent;
 
+import static net.runelite.api.ItemID.*;
+
 public class ClueScrollOverlay extends Overlay
 {
 	private static final ItemRequirement HAS_SPADE = new SingleItemRequirement(SPADE);
+	private static final int[] LIGHT_SOURCES = new int[]{LIT_CANDLE,CANDLE_LANTERN_4531, FIREMAKING_CAPE,
+			FIREMAKING_CAPET, BRUMA_TORCH, KANDARIN_HEADGEAR_1, KANDARIN_HEADGEAR_2, KANDARIN_HEADGEAR_3,
+			KANDARIN_HEADGEAR_4, MINING_HELMET_5014, BULLSEYE_LANTERN_4550, OIL_LANTERN_4539, OIL_LAMP_4524,
+			LIT_BLACK_CANDLE, SAPPHIRE_LANTERN_4702, EMERALD_LANTERN_20722, OIL_LAMP_4524, LIT_TORCH};
+	private static final ItemRequirement HAS_LIGHT = new VariableItemRequirement("LightSource", LIGHT_SOURCES);
```

Comment:
```java
import static net.runelite.client.plugins.cluescrolls.clues.item.ItemRequirements.item;
```

```suggestion
	private static final ItemRequirement HAS_LIGHT = new AnyRequirementCollection("Light Source", item(LIT_CANDLE), item(CANDLE_LANTERN_4531), item(FIREMAKING_CAPE), item(FIREMAKING_CAPET), item(BRUMA_TORCH), item(KANDARIN_HEADGEAR_1), item(KANDARIN_HEADGEAR_2), item(KANDARIN_HEADGEAR_3), item(KANDARIN_HEADGEAR_4), item(MINING_HELMET_5014), item(BULLSEYE_LANTERN_4550), item(OIL_LANTERN_4539), item(OIL_LAMP_4524), item(LIT_BLACK_CANDLE), item(SAPPHIRE_LANTERN_4702), item(EMERALD_LANTERN_20722), item(OIL_LAMP_4524), item(LIT_TORCH));

Reply:
Do the `FIREMAKING_CAPE` and `FIREMAKING_CAPET` vars also include the Max Cape and its variants, or were those accidentally forgotten about?

Reply:
Good spot. Only the regular max cape needs to be added though, as the variants don't act like skillcapes